src/frontend/src/components/edit-profile.js

```diff
@@ -531,168 +530,166 @@ class EditProfile extends PureComponent<Props> {
         <h2 className="form-screen__heading">{t("Profile.Title")}</h2>
         {errorMessage && <p className="account-profile__error">{errorMessage}</p>}
         <div className="form-screen__main">
-          {key && (
-            <div className="account-profile">
-              {!isAnonymous && (
-                <div className="account-profile__field">
-                  <label className="account-profile__label" htmlFor="headOfHousehold">
-                    {t("Accounts.Name")}
-                  </label>
+          <div className="account-profile">
```

Comment:
This is a minor thing, but when a formatter unexpectedly introduces changes like this that don't relate to the functional changes you're making, it's best to split out those changes into a separate commit as much as possible. This can make things easier on reviewers and later travelers who may be looking through commits to track down the source of a bug. I usually do this by using the "patch" feature of interactive-add: `git add -i`. It's not always easy to completely separate formatting from logic changes, but if you can get the bulk of it into a separate commit that's still helpful.

No need to go back and split these though, that's more trouble than it's worth at this point!